api/src/main/kotlin/edu/wgu/osmt/elasticsearch/SearchController.kt

```diff
@@ -142,6 +154,52 @@ class SearchController @Autowired constructor(
             .body(searchHits.map { it.content }.toList())
     }
 
+    @Transactional(readOnly = true)
+    @GetMapping(RoutePaths.EXPORT_SKILLS, produces = ["text/csv"])
+    @ResponseBody
+    fun scrollSkills(
```

Comment:
Would you please rename this method to exportLibrary? The idea of scrolling is very much an ElasticSearch implementation detail, and if we had a proper service layer, this controller would never know anything about scrolling vs searching vs whatever.